base/services/w32time/timedate.h

```diff
@@ -0,0 +1,124 @@
+#ifndef _TIMEDATE_H
+#define _TIMEDATE_H
+
+#include <stdarg.h>
+
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#define WIN32_NO_STATUS
+
+#include <windef.h>
+#include <winbase.h>
+#include <winnls.h>
+#include <winreg.h>
+#include <wingdi.h>
+#include <winuser.h>
+#include <wchar.h>
+#include <commctrl.h>
+#include <cpl.h>
+
+//#include "resource.h"
+
+#define MAX_KEY_LENGTH 255
+#define MAX_VALUE_NAME 16383
+#define SERVERLISTSIZE 6
+#define BUFSIZE 1024
+#define NTPPORT 123
+#define ID_TIMER 1
+
+typedef struct
+{
+  UINT idIcon;
+  UINT idName;
+  UINT idDescription;
+  APPLET_PROC AppletProc;
+} APPLET, *PAPPLET;
```

Comment:
2-space indenting

Reply:
This was copied exactly from dll/cpl/timedate/timedate.h. Should I change it here?